aws-ts-vpc-with-ecs-fargate-py/ecs-fargate-python/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Please can we remove the LICENSE file here - this isn't something we tend to do to these examples

Reply:
Removed